samples/paved-path-demo/paved-path-demo-promise.yaml

```diff
@@ -183,3 +183,5 @@ spec:
             containers:
               - image: syntasso/paved-path-demo-configure-pipeline
                 name: paved-path-demo-configure-pipeline
+            Tolerations:
+            - operator: Exists
```

Comment:
```suggestion
```
Thanks for fixing the promise name! Could you remove the toleration from this example promise? Workflow configurations are usually documented in: https://docs.kratix.io/main/reference/workflows#pipelines I could add it after we merge this PR.